gradle.properties

```diff
@@ -4,12 +4,12 @@ org.gradle.daemon=false
 
 # Minecraft and Forge
 minecraft_version=1.20.1
-forge_version=47.1.0
+forge_version=47.1.1
```

Comment:
should we consider to keep minimum forge version the same as dev forge version to make sure that we don't use newer features than our version range requires?

Reply:
See that's something we *should* do, but then that dismantles the point of the PR :P. I guess its the kind of thing that'd be a pain in the butt to keep on top of tbh though so might be sensible to not take this course of action. Up to you guys :)

Reply:
we could set our dev version to 47.0.34 (that's what I was advocating for if we reduce the min forge version)

Reply:
Ah right I got you; i just thought it'd be nice to continue to have bug fixes in dev, but maybe it is more sensible

Reply:
if we go back, can we go back to at most 47.0.40? As if we qould require this, then the weird nullable check I added for menu opening of the photovoltaic cells wouldn't be required anymore